Win32/Prj/libpcap.vcxproj

```diff
@@ -109,26 +113,41 @@
       <SuppressStartupBanner>true</SuppressStartupBanner>
       <OutputFile>.\Debug\libpcap.lib</OutputFile>
     </Lib>
+    <Link>
+      <AdditionalDependencies>ws2_32.lib;..\..\..\..\packetWin7\Dll\Project\Release No NetMon and AirPcap\Packet.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClCompile Include="..\..\bpf\net\bpf_filter.c" />
     <ClCompile Include="..\..\bpf_dump.c" />
     <ClCompile Include="..\..\bpf_image.c" />
     <ClCompile Include="..\..\etherent.c" />
+    <ClCompile Include="..\..\fad-helpers.c" />
     <ClCompile Include="..\..\fad-win32.c" />
     <ClCompile Include="..\..\gencode.c" />
-    <ClCompile Include="..\..\grammar.c" />
+    <ClCompile Include="..\..\..\Win32-Extensions\grammar.c" />
     <ClCompile Include="..\..\inet.c" />
     <ClCompile Include="..\..\nametoaddr.c" />
     <ClCompile Include="..\..\optimize.c" />
+    <ClCompile Include="..\..\pcap-common.c" />
     <ClCompile Include="..\..\pcap-tc.c" />
     <ClCompile Include="..\..\Pcap-win32.c" />
     <ClCompile Include="..\..\pcap.c" />
     <ClCompile Include="..\..\savefile.c" />
-    <ClCompile Include="..\..\scanner.c" />
+    <ClCompile Include="..\..\..\Win32-Extensions\scanner.c" />
+    <ClCompile Include="..\..\sf-pcap-ng.c" />
+    <ClCompile Include="..\..\sf-pcap.c" />
   </ItemGroup>
   <ItemGroup>
+    <ClInclude Include="..\..\..\Win32-Extensions\bitset.h" />
+    <ClInclude Include="..\..\..\Win32-Extensions\grammar.h" />
+    <ClInclude Include="..\..\..\Win32-Extensions\scanner.h" />
+    <ClInclude Include="..\..\..\Win32-Extensions\TcApi.h" />
+    <ClInclude Include="..\..\..\Win32-Extensions\tokdefs.h" />
+    <ClInclude Include="..\..\pcap-common.h" />
```

Comment:
Again, the build process should run Flex and Bison, which should put those source files in the main source directory, so that change shouldn't be necessary.